worker-core/src/main/java/com/github/workerframework/core/WorkerThreadPool.java

```diff
@@ -61,6 +61,8 @@ void awaitTermination(long timeout, TimeUnit unit)
     boolean isIdle();
 
     int getBacklogSize();
+    
+    int getActiveCount();
```

Comment:
I see now that this is only used for logging.  Maybe it would be better to call it `getApproxActiveCount()` just to emphasize that the result does not impact functionality?